packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -83,43 +86,70 @@ export const buildBreadcrumbManager = (
   function getFacetBreadcrumbs(): FacetBreadcrumb[] {
     return getBreadcrumbs<FacetValue>(
       engine.state.facetSet,
-      executeToggleFacetSelect,
+      ({facetId, selection}) => {
+        const analyticsAction = logFacetBreadcrumb({
+          facetId: facetId,
+          facetValue: selection.value,
+        });
+        dispatch(toggleSelectFacetValue({facetId, selection}));
+        dispatch(
+          updateFreezeCurrentValues({facetId, freezeCurrentValues: false})
+        );
```

Comment:
I'm a bit uncertain. Why dispatch this here but not in the other facet getters? (e.g. `getNumericFacetBreadcrumbs`)

Reply:
Good question!
Because ranges do not need to freeze current values (the values are always in the same order) and for the category, we deselect all the values which already sets freezeCurrentValues to false. Only in the normal facet do we set it to true when toggling a value